data/translations/da.ts

```diff
@@ -45,10 +49,46 @@
         <source>Enter your username and password</source>
         <translation>Indtast dit brugernavn og adgangskode</translation>
     </message>
+    <message>
+        <source>Reboot</source>
+        <translation>Genstart</translation>
+    </message>
+    <message>
+        <source>Session</source>
+        <translation>Session</translation>
+    </message>
+    <message>
+        <source>Shutdown</source>
+        <translation>Luk ned</translation>
+    </message>
+    <message>
+        <source>User name</source>
+        <translation>Brugernavn</translation>
+    </message>
     <message>
         <source>Select your user and enter password</source>
         <translation>Vælg din bruger og indtast adgangskode</translation>
     </message>
+    <message>
+        <source>Enter your username</source>
+        <translation>Indtast dit brugernavn</translation>
+    </message>
+    <message>
+        <source>Enter your password</source>
+        <translation>Indtast din adgangskode</translation>
+    </message>
+    <message>
+        <source>Suspend</source>
+        <translation>Hvile</translation>
+    </message>
+    <message>
+        <source>Hibernate</source>
+        <translation>Dvale</translation>
+    </message>
+</context>
+</TS>
```

Comment:
why does it end <TS> here?

Reply:
I based it on ru.ts.
Did not generate the file my self since this is difficult.
How else should it end?